specification/elastic/resource-manager/Microsoft.Elastic/stable/2023-06-01/elastic.json

```diff
@@ -0,0 +1,2510 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/VMIngestionDetailsResponse"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "VMIngestion_Details": {
+            "$ref": "./examples/VMIngestion_Details.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/vmCollectionUpdate": {
+      "post": {
+        "tags": [
+          "VMCollectionUpdate"
+        ],
+        "summary": "Update the vm details that will be monitored by the Elastic monitor resource.",
+        "operationId": "VMCollection_Update",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "VM resource Id",
+            "schema": {
+              "$ref": "#/definitions/VMCollectionUpdate"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "VMCollection_Update": {
+            "$ref": "./examples/VMCollection_Update.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Elastic/monitors/{monitorName}/listUpgradableVersions": {
+      "post": {
+        "tags": [
+          "MonitorUpgradableVersions"
+        ],
+        "summary": "List of upgradable versions for a given monitor resource.",
+        "operationId": "UpgradableVersions_Details",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Pagination Support?

Reply:
@visingla-ms 
As discussed offline, we're just creating a stable version for the preview API versions which are already available and no new changes are being pushed here.
PFB the previous PRs:
https://github.com/Azure/azure-rest-api-specs/pull/22638
https://github.com/Azure/azure-rest-api-specs/pull/23343
https://github.com/Azure/azure-rest-api-specs/pull/23602
https://github.com/Azure/azure-rest-api-specs/pull/23650
https://github.com/Azure/azure-rest-api-specs/pull/24031